### ISSUE TITLE
Use `<ol>`/`<ul>` for lists in tables

### ISSUE THREAD

**Issue by 0x009922** [fraud-intelligence-limited/fil-documentation#3]
## Issue example

```md
Could be one of the following: <br> 1. A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code. <br> 2. A range of **IP addresses** (or a single one). <br> 3. A unique **IMEI identifier** of a device.
```

> Could be one of the following: <br> 1. A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code. <br> 2. A range of **IP addresses** (or a single one). <br> 3. A unique **IMEI identifier** of a device.

## How it should be

The list should be rendered not as `1. xx <br> 2. yy`, but as `<ol><li>xx</li><li>yy</li></ol>`

```md
Could be one of the following: <ol> <li>A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code. </li> <li>A range of **IP addresses** (or a single one).</li> <li>A unique **IMEI identifier** of a device.</li></ol>
```

> Could be one of the following: <ol> <li>A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code. </li> <li>A range of **IP addresses** (or a single one).</li> <li>A unique **IMEI identifier** of a device.</li></ol>

**Comment by 0x009922** [comment]
@6r1d,

Also, consider the possibility to move highly-repetitive parts (e.g. description of the `fraudType` field) into re-usable Vue component.

- [Using Vue in Markdown > Using Components | VitePress](https://vitepress.dev/guide/using-vue#using-components)
- [Extending the Default Theme > Registering Global Components | VitePress](https://vitepress.dev/guide/extending-default-theme#registering-global-components)

In case of the `fraudType`, the component might look somehow like this:

```vue
<template>
  The type of the fraud event. <br />
  Could be one of the following:
  <ol>
    <li><code>Wangiri</code></li>
    <li><code>IRSF</code></li>
    <li><code>StolenDevice</code></li>
    <li><code>IPFraud</code></li>
    <li><code>SMSA2P</code></li>
  </ol>
</template>
```

**Comment by yamkovoy** [comment]
Resolved in pull request #32
